### ISSUE TITLE
What kind of UI component should be used to change the magnet orientation?

### ISSUE THREAD

**Issue by samreid** [phetsims/faradays-law#100]
From https://github.com/phetsims/faradays-law/issues/98

> For the radio buttons, we should discuss with Taliesin whether it should be treated as togglebutton/checkbox/radio buttons for accessibility. We can use that as guidance for how to PhET-iO instrument it.  AP thinks it will be a toggle or checkbox, not radio buttons.

Currently there is a rectangular pushbutton to flip the magnet. It seems convenient that you don't need to move the mouse to flip it multiple times (as you would with a traditional radio button).  @terracoda @jessegreenberg and @emily-phet can you please comment what scenery/a11y peer component type you would recommend for this user interface component, or if rectangular pushbutton is already appropriate?

**Comment by jessegreenberg** [comment]
I agree, the RectangularPushButton that is already there seems great, but it would be good to hear from @terracoda. @terracoda for a11y would the "flip magnet" button in this sim need to be anything other than a `<button>` in the parallel DOM?
![capture](https://user-images.githubusercontent.com/6396244/34421369-1ded0da0-ebcc-11e7-98de-9d19654f99bc.PNG)

**Comment by jessegreenberg** [comment]
We discussed in 1/2/18 meeting, and agreed that this button doesn't need to be changed for a11y purposes and a RectangularPushButton is appropriate. It will probably be a `<button>`, with some kind of alert confirmation that the flip happened after button was pressed like "Magnet flipped to north-south orientation."

Back to @samreid for review or further questions.

**Comment by samreid** [comment]
Sounds good, thanks @jessegreenberg and @terracoda, closing.
